include/s_newconf.h

```diff
@@ -157,6 +157,7 @@ extern void cluster_generic(struct Client *, const char *, int cltype,
 #define IsOperInvis(x)          (HasPrivilege((x), "oper:hidden"))
 #define IsOperRemoteBan(x)      (HasPrivilege((x), "oper:remoteban"))
 #define IsOperMassNotice(x)     (HasPrivilege((x), "oper:mass_notice"))
+#define IsOperWallOps(x)        (HasPrivilege((x), "oper:wallops"))
```

Comment:
There's no need to continue the `IsOper*` tradition, imo; I'd just use HasPrivilege in the conditional directly.
```suggestion
```